src/GovUk.Education.ExploreEducationStatistics.Content.Api/ViewModels/AllMethodologiesViewModels.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model.ViewModels;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.ViewModels
+{
+    public class AllMethodologiesThemeViewModel : IComparable<AllMethodologiesThemeViewModel>
+    {
+        public Guid Id { get; set; }
+        public string Title { get; set; }
+        public List<AllMethodologiesTopicViewModel> Topics { get; set; }
+
+        public void RemoveTopicLeafNodesAndSort()
```

Comment:
Might consider just renaming this method slightly to indicate which leaf nodes we'd be removing e.g.

`RemoveTopicNodesWithoutMethodologiesAndSort`

Also below:

`RemovePublicationNodesWithoutMethodologiesAndSort`

The fact that they're leaf nodes didn't immediately occur to me that that meant they didn't have Methodologies under them.

Reply:
Thanks, I've renamed them now.